### PR TITLE
Shuttle stats

### PR DESCRIPTION
**Pull request by trv48226**
## Description

I added a template to later add shuttle stats for students. Also linked the navbar but that should be updated to not have a blue line under it. I do repeat some code so maybe I should change some parts?

Fixes # (issue)

## Size and Type of change

Delete any of these you don't use.

- Small Change - 1 person needs to review this

- New feature

## How Has This Been Tested?

Please briefly describe the tests that you ran to verify your changes.

I looked at it on different browsers and it looks fine

## Checklist:

Only check what applies and what you have done.

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have tested on multiple browsers (Chrome, Firefox, Safari, IE suite)

### CODE REVIEW

File: shuttle/templates/nav.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <nav class="navbar navbar-expand-lg navbar-dark background-blue">
     <a class="navbar-brand" href="{{ url_for('View:index') }}">
         <img height="48" src="https://confluence.bethel.edu/download/attachments/327682/global.logo?version=4&amp;modificationDate=1462424762000&amp;api=v2">
-        <span class="ml-2  h6 lab-title">Shuttle</span>
+        <a href="/"><span class="ml-2  h6 lab-title" style="color:white">Shuttle</span></a>
+        <a href="/shuttle-stats"><span class="ml-2  h6 lab-title background-blue" style="color:white">Shuttle Stats</span></a>
```
Comment (bam95899) [inline shuttle/templates/nav.html:5]: Change the href to use url_for just like on line 2 of this file.

Comment (trv48226) [reply shuttle/templates/nav.html]: Ahh good idea. I changed those

File: shuttle/templates/shuttle_stats.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <div class="col">
+            <div class="card" >
+                <div class="card-body">
+                    <h4 class="card-title">Shuttle Schedule</h4><hr>
+                    <h5>*Insert schedule here*</h5>
+                </div>
+            </div>
+        </div>
+    </div>
+{% endblock %}
```
Comment (bam95899) [inline shuttle/templates/shuttle_stats.html:36]: Could you move your html file inside a new directory inside of templates called `schedules`. So the path would be /templates/schedules/shuttle_stats.html.

File: shuttle/views.py [hunk 3]
```diff
@@ -9,6 +9,10 @@ class View(FlaskView):
     def index(self):
         return render_template('index.html')
 
+    @route('/shuttle-stats')
+    def stats(self):
+        return render_template('shuttle_stats.html')
+
```
Comment (bam95899) [inline shuttle/views.py:15]: Could you move this route inside of a new init file and put it in a new directory called `schedules`. The path would be /shuttle/schedules/\_\_init\_\_.py

Comment (bam95899) [reply shuttle/views.py:15]: Set it up similarly to this file https://github.com/betheluniversity/writing-center-python/blob/master/writing_center/appointments/__init__.py 

Basically you need to create a class called `SchedulesView` and then put the route in it.

Comment (bam95899) [reply shuttle/views.py:15]: You would then have to register the view in the base level init. Check in the current base level init to see how View is imported and registered.

Comment (trv48226) [reply shuttle/views.py]: Sounds good. I just made those changes

**Pull request merged**